app/src/main/res/layout/fragment_signin.xml

```diff
@@ -1,7 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Daha önce de konuşmuştuk, tekrar not olması için yazıyorum. ScrollView alınması iyi olabilr.